mvpa2/tests/test_procrust.py

```diff
@@ -105,7 +105,10 @@ def test_simple(self, svd, oblique):
 
                 # Test if we get back
                 d_s_f_r = pm.reverse(d_s_f)
-
+                # Test if recon proj is true inverse except for high->low projection
+                if nf_s <= nf_t:
+                    self.assertTrue(norm(np.dot(pm._proj, pm._recon) - np.eye(pm._proj.shape[0])) < 1e-10,
```

Comment:
ok -- make it kosher here (`assert_array_almost_equal` instead of norm etc) and we should be done ;)